src/reducers/bloodlinesReducer.js

```diff
@@ -1,53 +1,48 @@
 import {
-	GET_ALL_CONTENT,
-	HANDLE_GET_ALL_CONTENT,
-	ERROR_GET_ALL_CONTENT,
+	HANDLE_PAGED,
+	ERROR_PAGED,
 
-	GET_CONTENT,
-	HANDLE_GET_CONTENT,
-	ERROR_GET_CONTENT,
+	TRIGGERS,
+	CONTENTS,
 
-	CREATE_CONTENT,
-	HANDLE_CREATE_CONTENT,
-	ERROR_CREATE_CONTENT,
-
-	DELETE_CONTENT,
-	HANDLE_DELETE_CONTENT,
-	ERROR_DELETE_CONTENT,
-
-	GET_ALL_TRIGGERS,
-	HANDLE_GET_ALL_TRIGGERS,
-	ERROR_GET_ALL_TRIGGERS,
-
-	GET_TRIGGER,
-	HANDLE_GET_TRIGGER,
-	ERROR_GET_TRIGGER,
-
-	CREATE_TRIGGER,
-	HANDLE_CREATE_TRIGGER,
-	ERROR_CREATE_TRIGGER,
-
-	DELETE_TRIGGER,
-	HANDLE_DELETE_TRIGGER,
-	ERROR_DELETE_TRIGGER
+	REQUEST,
+	HANDLE,
+	ERROR,
+	TIMEOUT
 } from '../actions/bloodlinesActions'
 
-export function getAllContent(state = {
+export function triggers(state = {
+	fetching: false,
+	cursor: 0,
+	next: false,
+	items: {},
+	ids: [],
+	error: null,
+}, action) {
+	if (action.itemType !== TRIGGERS) {
```

Comment:
This is the thing i'm most uncertain about, but i wanted to consolidate the below function while still returning separate states for the different objects so we can access all triggers and all contents in their respective containers.